VERSION

```diff
@@ -1 +1 @@
-0.6.6-dev
```

Comment:
You don't need to bump the release version here, as you are not making any changes to the golang code.

Reply:
> You don't need to bump the release version here, as you are not making any changes to the golang code.

Oh ok. I just reverted since there is no change on go files.